src/cosmos/faucet/discord_message.rs

```diff
@@ -0,0 +1,109 @@
+//! Implementation of a faucet discord message result for after transaction
+
+use crate::cosmos::tx::discord_message::TransactionDiscordMessage;
+use crate::cosmos::tx::error::Error as TxError;
+use crate::discord::discord_client::message::DiscordMessage;
+use cosmos_sdk_proto::cosmos::base::abci::v1beta1::TxResponse;
+use serenity::async_trait;
+use serenity::http::Http;
+use serenity::model::channel::Message;
+use serenity::model::id::ChannelId;
+use serenity::model::user::User;
+use serenity::prelude::Mentionable;
+use serenity::Error;
+use tracing::{error, info};
+
+/// The faucet respons emessage when transaction successful
```

Comment:
```suggestion
/// Faucet transaction response
```
(there's a message even if the transaction failed)